XmlFormatter/src/Windows/MainForm.cs

```diff
@@ -90,6 +103,58 @@ private void MainForm_Load(object sender, EventArgs e)
             {
                 CheckForUpdatedVersion(true);
             }
+
+            SetFormatter(new XmlFormatterProvider());
+            SetupHotFolder();
+        }
+
+        private void SetupHotFolder()
+        {
+            if (!Properties.Settings.Default.HotfolderActive)
+            {
+                if (hotfolderManager != null)
+                {
+                    hotfolderManager.ResetManager();
+                    hotfolderManager = null;
+                }
+                
+                return;
+            }
+            if (hotfolderManager == null)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '??=' operator here.](https://app.codacy.com/manual/simonaberle/XmlFormatter/pullRequest?prid=5053278)